packages/acceptance-tests/pkg-tests-specs/sources/commands/__snapshots__/workspace.test.js.snap

```diff
@@ -0,0 +1,23 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Commands workspace <workspace-name> <sub-command> runs a given command in the specified workspace 1`] = `
+Object {
+  "code": 1,
+  "stderr": "",
+  "stdout": "Usage Error: Couldn't find a script named \\"workspace package-a print\\".
```

Comment:
I don't know how I tell the test to pick up the new command. I copied the `.yarnrc.yml` setup from `foreach.test.js` which is part of the same plugin as far as I can tell.

Reply:
It's because you used `run('foo bar baz')` instead of `run('foo', 'bar', 'baz')`